mlir/lib/Dialect/Rock/Generator/Conv2dGenerator.cpp

```diff
@@ -266,6 +266,11 @@ LogicalResult Conv2dGenerator::hasValidChip() const {
   if (bitEnumContainsAll(config.features, GemmFeatures::mfma) &&
       (chipHexNumber != 0x908 && chipHexNumber != 0x90a))
     return failure();
+
+  // WMMA is only supported on gfx1100
```

Comment:
Hold on a minute, WMMA is on all gfx11xx, let's support all of those preemptively.